core/ingest/README.md

```diff
@@ -0,0 +1,69 @@
+# Polymath Ingest
+
+This is a tool that is used to ingest data from a source and convert it into a format that can be used by the Polymath system.
+
+## Standalone ingestion tool
```

Comment:
Can this go into `core/cli`? In other words, most of the importing code stays here, but the actual invocation is made using `polymath` tool. I'd love to keep only one CLI entry point for Polymath.

Reply:
I was thinking about this a bit the other day and I was worrying that the base CLI would get incredibly complex... however, I could do an "import" argument.